deps/patches/libgit2-ssh.patch

```diff
@@ -1,23 +0,0 @@
---- CMakeLists.txt	2016-05-26 23:13:48.000000000 -0400
```

Comment:
This is the only one I'm not sure about. I don't quite understand what this patch is trying to do, and the CMake files have changed considerably (some of this has been moved to `src/CMakeLists.txt`).

Reply:
windows ssh clones. test a source build or clean from the buildbot, or this may have broken libgit2 linking to libssh2 and working on private repos